src/plugins/telemetry/telemetry_impl.cpp

```diff
@@ -2138,6 +2263,14 @@ void TelemetryImpl::subscribe_heading(Telemetry::HeadingCallback& callback)
     _heading_subscription = callback;
 }
 
+void TelemetryImpl::request_home_position_async()
+{
+    MavlinkCommandSender::CommandLong command_request_message{*_parent};
+    command_request_message.command = MAV_CMD_REQUEST_MESSAGE;
+    command_request_message.params.param1 = MAVLINK_MSG_ID_HOME_POSITION;
+    _parent->send_command_async(command_request_message, nullptr);
+}
+
```

Comment:
Where is the response to that request used? I think I'm missing it :thinking:

Reply:
We setup a message handler for the `HOME_POSITION` message here https://github.com/mavlink/MAVSDK/blob/634f171bb5b492e2d5bd6453f54e7e88c603899e/src/plugins/telemetry/telemetry_impl.cpp#L42-L45  which is used to handle the response.